### PR TITLE
feature/BACKEND-0001 "Add initial config for project "

### PR DESCRIPTION
**Pull request by cristiantcjti**

### CODE REVIEW

File: .idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<component name="InspectionProjectProfileManager">
```
Comment (PedroTeixeira-dev) [inline .idea/inspectionProfiles/Project_Default.xml:1]: @cristiantcjti i Belize that this file was meant to be in the .gitignore file, please review

File: Dockerfile [hunk 2]
```diff
@@ -25,9 +25,9 @@ RUN pipenv install --system --deploy
 
 COPY . ${APP_HOME}
 
-EXPOSE 80
+EXPOSE 5000
```
Comment (PedroTeixeira-dev) [inline Dockerfile:28]: @cristiantcjti the expose address was changed from 80 to 5000

File: Dockerfile [hunk 3]
```diff
@@ -25,9 +25,9 @@ RUN pipenv install --system --deploy
...
 
 RUN chmod ugo+rwx -R /app
 
 CMD /bin/sh -c "flask db upgrade -d back-end/migrations; \
-  gunicorn --config gunicorn_config.py wsgi:application"
\ No newline at end of file
+  gunicorn --config wsgi:application"
```
Comment (PedroTeixeira-dev) [inline Dockerfile:33]: @cristiantcjti the gunicorn configuration was changed, by adding the unicorn_config.py file in the command line

File: Pipfile [hunk 4]
```diff
@@ -4,16 +4,48 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
```
Comment (PedroTeixeira-dev) [inline Pipfile:6]: @cristiantcjti the versions of the dependences were added. Other dependences were added as well.

File: Dockerfile (hunk 2 above)
Comment (cristiantcjti) [reply Dockerfile:28]: @PedroTeixeira-dev Exactly, that is the right port to be the project

**Pull request closed without merging**